ngs_mapper/runsample.py

```diff
@@ -440,25 +441,45 @@ def select_keys(d, keys):
                 logger.critical( "{0} failed to complete sucessfully. Please check the log file {1} for more details".format(cmd,bwalog) )
                 sys.exit(1)
 
-        # Tag Reads
-        cmd = 'tagreads {bamfile} -CN {CN}'
-        if cmd_args['config']:
-            cmd += ' -c {config}'
-        p = run_cmd( cmd.format(**cmd_args), stdout=lfile, stderr=subprocess.STDOUT )
+        tcmd = "samtools view -F 2048 -b {tmp_bam} > {bamfile}"
```

Comment:
You might want to remove the tmp_bam file to be considerate of space or make that an option